qa/suites/fs/volumes/tasks/volumes/test/emptystring.yaml

```diff
@@ -0,0 +1,5 @@
+tasks:
+  - cephfs_test_runner:
+      fail_on_skip: false
+      modules:
```

Comment:
Also this commit can be squashed with the one that adss tests.

Reply:
squashed the commits